Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -32,6 +32,18 @@ function Get-xSharePointContentService() {
     return [Microsoft.SharePoint.Administration.SPWebService]::ContentService
 }
 
+
+function Get-xSharePointUserProfileSubTypeManager {
+    [CmdletBinding()]
+    param
+    (
+        $context
```

Comment:
Traviz. indeed, we are being inconsistent here :(
I will address this very one but I think we need a unit test to validate parameters are in the correct case. 
@BrianFarnhill  we might need to add this test in our 'Technical debt' backlog.